cxx4opencl/intro.txt

```diff
@@ -7,7 +7,12 @@
 
 This language is built on top of OpenCL C v2.0 and {cpp}17 enabling
 most of regular {cpp} features in OpenCL kernel code. Most functionality
-from OpenCL C and {cpp} is inherited.
+from OpenCL C and {cpp} is inherited. Since both OpenCL C and {cpp} are
+derived from C and moreover {cpp} is backward compatible to C, the main
```

Comment:
```suggestion
derived from C and moreover {cpp} is backward compatible with C, the main
```

Reply:
```suggestion
derived from C and moreover {cpp} is almost fully backward compatible to C, the main
```

Reply:
You did not integrate the second suggestion.
(perhaps I should have merged them into one)

Reply:
Ah strange! Not sure what went wrong...